backend/Virkarekisteri/src/Functions/OrganizationTree/GetOrganizationTrees.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Extensions.Logging;
+using Virkarekisteri.Repositories;
+
+namespace Virkarekisteri.Functions.OrganizationTree
+{
+    public class GetOrganizationTrees
+    {
+        private readonly ILogger<GetOrganizationTrees> _logger;
+        private readonly IOrganizationTreeRepository _organizationTreeRepository;
+
+        public GetOrganizationTrees(
```

Comment:
Would be nice if we utilized primary constructors here as well to keep the functions in line with all current other functions.

Reply:
Fixed